src/dmd/typesem.d

```diff
@@ -3745,7 +3745,8 @@ Expression dotExp(Type mt, Scope* sc, Expression e, Identifier ident, int flag)
      *  sc = context
      *  e = `this` for `ident`
      *  ident = name of member
-     *  flag = if flag & 1, don't report "not a property" error and just return NULL.
+     *  flag = flag & 1, don't report "not a property" error and just return NULL.
```

Comment:
Isn't `1` actually `DotExpFlag.gag` ?

Reply:
Yes, i'm going to change every '1' in another commit.